creme/creme_config/static/creme_config/js/menu-brick.js

```diff
@@ -1,3 +1,5 @@
+creme.creme_config = creme.creme_config || {};
+
 creme.creme_config.MenuContainersController = creme.component.Component.sub({
```

Comment:
You'll want to move creme.creme_config.MenuContainersController to creme.MenuContainersController in a future patch ?

Reply:
yes